### ISSUE TITLE
Some ext-sodium functions are missing from polyfill

### ISSUE THREAD

**Issue by bburnichon** [paragonie/sodium_compat#92]
I have a PHP project which is not (yet) compatible with PHP7.2+ versions but that uses lib sodium.

I wish to use timing attacks safer versions of several functions like `bin2base64` which are declared in extension from version 7.2.0 https://github.com/php/php-src/blob/PHP-7.2.0/ext/sodium/php_libsodium.h#L39
Sadly, some of these are not implemented in this repo. So this repo can NOT say it provides `ext-sodium` as is intended in https://github.com/paragonie/sodium_compat/pull/59#issuecomment-363790988

**Comment by paragonie-scott** [comment]
Those functions in libsodium are actually *newer* than the polyfill lib. I'll get to work on them for `v1.11.0`.

**Comment by szepeviktor** [comment]
Should we include a CI step that checks existence all functions over at ext-sodium?

**Comment by paragonie-scott** [comment]
We've got a full list now.

```
add: NOT FOUND
base642bin: NOT FOUND
bin2base64: NOT FOUND
crypto_kdf_derive_from_key: NOT FOUND
crypto_kdf_keygen: NOT FOUND
crypto_kx_client_session_keys: NOT FOUND
crypto_kx_keypair: NOT FOUND
crypto_kx_publickey: NOT FOUND
crypto_kx_secretkey: NOT FOUND
crypto_kx_seed_keypair: NOT FOUND
crypto_kx_server_session_keys: NOT FOUND
crypto_pwhash_str_needs_rehash: NOT FOUND
crypto_secretstream_xchacha20poly1305_keygen: NOT FOUND
crypto_secretstream_xchacha20poly1305_init_push: NOT FOUND
crypto_secretstream_xchacha20poly1305_push: NOT FOUND
crypto_secretstream_xchacha20poly1305_init_pull: NOT FOUND
crypto_secretstream_xchacha20poly1305_pull: NOT FOUND
crypto_secretstream_xchacha20poly1305_rekey: NOT FOUND
crypto_sign_keypair_from_secretkey_and_publickey: NOT FOUND
pad: NOT FOUND
unpad: NOT FOUND
```

**Comment by paragonie-scott** [comment]
Fixed in `v1.11.0`
